### ISSUE TITLE
Owner still not correct

### ISSUE THREAD

**Issue by helendallas** [kartoza/django-bims#2440]
I have been checking why my data capturer's are assigned as owner instead of the scientists that did the SASS assessment.

https://freshwaterbiodiversity.org/sass/dashboard-multi-sites/?taxon=&search=&siteId=&collector=%5B%22Janette%20Lumley%22%5D&category=&yearFrom=&yearTo=&months=&boundary=&userBoundary=&referenceCategory=&spatialFilter=&reference=&endemic=&conservationStatus=[]&modules=&validated=&sourceCollection=[%22fbis%22]&abioticData=&ecologicalCategory=&rank=&siteIdOpen=&orderBy=name&polygon=

This query was filter = Janette Lumley. 
![image](https://user-images.githubusercontent.com/45286876/101348798-ebf5c580-3894-11eb-99b7-b184b180c06c.png)

But the csv file shows all owners are correct as per the study.  I think that the Data capturer is being used instead of Collector/Owner 

![image](https://user-images.githubusercontent.com/45286876/101348684-b7820980-3894-11eb-96e5-ccb1545b9259.png)

[FBIS data extrcated for Filter Collector_Owner= Janette Lumley.xlsx](https://github.com/kartoza/django-bims/files/5652509/FBIS.data.extrcated.for.Filter.Collector_Owner.Janette.Lumley.xlsx)

Link to #2380

**Comment by dimasciput** [comment]
@helendallas yes it's true, the filter are based on **collector** or **owner**. 
![image](https://user-images.githubusercontent.com/1979569/101445547-46099000-3954-11eb-92c6-2d8bd07f304f.png)

Should I change the filter from collector/owner to just owner instead?

**Comment by dimasciput** [comment]
In our system **collector** is the same with **capturer**

**Comment by helendallas** [comment]
@dimasciput Okay in that case we only want to see the **Owner**, not collector/capturer.  Please can you make a note to revise.
This is the person who owns the data and is able to edit it.

**Comment by helendallas** [comment]
Yes, for above filter, perhaps remove collector, and only include owner, thanks

**Comment by dimasciput** [comment]
@helendallas I think fixed this one, can you try it again?

**Comment by helendallas** [comment]
@dimasciput Checked and fine now, thanks Closing ticket
